client-skeleton-consumer/src/main/java/eu/arrowhead/client/skeleton/consumer/ConsumerMain.java

```diff
@@ -0,0 +1,76 @@
+package eu.arrowhead.client.skeleton.consumer;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+
+import eu.arrowhead.client.skeleton.common.ArrowheadService;
+import eu.arrowhead.common.CommonConstants;
+import eu.arrowhead.common.dto.shared.OrchestrationFlags.Flag;
+import eu.arrowhead.common.dto.shared.OrchestrationFormRequestDTO;
+import eu.arrowhead.common.dto.shared.OrchestrationFormRequestDTO.Builder;
+import eu.arrowhead.common.dto.shared.OrchestrationResponseDTO;
+import eu.arrowhead.common.dto.shared.OrchestrationResultDTO;
+import eu.arrowhead.common.dto.shared.ServiceQueryFormDTO;
+import eu.arrowhead.common.exception.ArrowheadException;
+
+@SpringBootApplication
+@ComponentScan(basePackages = {CommonConstants.BASE_PACKAGE}) //TODO: add custom packages if any
+public class ConsumerMain implements ApplicationRunner {
+    public static void main( final String[] args ) {
+    	SpringApplication.run(ConsumerMain.class, args);
+    }
+    
+    //-------------------------------------------------------------------------------------------------
+    @Autowired
+	private ArrowheadService arrowheadService;
+    
+ 	//-------------------------------------------------------------------------------------------------
+    @Override
+	public void run(final ApplicationArguments args) throws Exception {
+		//SIMPLE EXAMPLE OF INITIATING AN ORCHESTRATION
+    	
+    	final Builder orchestrationFormBuilder = arrowheadService.getOrchestrationFormBuilder();
+    	
+    	final ServiceQueryFormDTO requestedService = new ServiceQueryFormDTO();
+    	requestedService.setServiceDefinitionRequirement("test-service");
+    	
+    	orchestrationFormBuilder.requestedService(requestedService)
+    							.flag(Flag.MATCHMAKING, false) //When this flag is false or not specified, then the orchestration response cloud contain more proper provider. Otherwise only one will be chosen if there is any proper.
+    							.flag(Flag.EXTERNAL_SERVICE_REQUEST, false) //When this flag is false or not specified, then the orchestration will look for proper providers only within the local cloud. Otherwise orchestration will look for providers only in the neighbor clouds.
```

Comment:
this is totally wrong, delete it (not just the comment, the whole flag setting). EXTERNAL_SERVICE_REQUEST is for core systems only.